pkg/controllers/scheduler/framework/plugins/rsp/rsp_test.go

```diff
@@ -442,6 +442,13 @@ func TestClusterWeights(t *testing.T) {
 			schedulingUnit: framework.SchedulingUnit{
 				DesiredReplicas: pointer.Int64(10),
 				SchedulingMode:  fedcorev1a1.SchedulingModeDivide,
+				CustomMigration: framework.CustomMigrationSpec{
```

Comment:
Why add `CustomMigration` here if it doesn't have any effect?

Reply:
This is a mistake.